packages/server/git-repo-info.js

```diff
@@ -0,0 +1,13 @@
+const gitRepoInfo = require('git-repo-info')
+const fs = require('fs')
+
+const info = gitRepoInfo()
+const version = require('./package.json').version
+
+fs.writeFile('repo-info.json', JSON.stringify({ ...info, version }), (err, info) => {
```

Comment:
```suggestion
fs.writeFileSync('repo-info.json', JSON.stringify({ ...info, version }), (err, info) => {
```

Should be `writeFileSync` to ensure the file is ready when you start the server